sdk/identity/Azure.Identity/src/ImdsManagedIdentitySource.cs

```diff
@@ -63,7 +65,11 @@ protected override Request CreateRequest(string[] scopes)
 
             if (!string.IsNullOrEmpty(_clientId))
             {
-                request.Uri.AppendQuery("client_id", _clientId);
+                request.Uri.AppendQuery(Constants.ManagedIdentityClientId, _clientId);
+            }
+            if (!string.IsNullOrEmpty(_resourceId))
```

Comment:
Is it possible both _clientId and _resourceId are not empty? I think the answer is no?

Reply:
I believe that if neither is provided this would indicate that the system assigned identity would be used.